include/default-opts/lwipopts.h

```diff
@@ -44,6 +44,24 @@
 #define PBUF_DEBUG LWIP_DBG_OFF
 #define SOCKETS_DEBUG LWIP_DBG_ON
 #endif
+
+#define LWIP_STATS 0
```

Comment:
Maybe
```suggestion
#ifndef LWIP_STATS
#define LWIP_STATS 0
#endif
```
so we can enable it without modifying the code?